.github/workflows/schemacode_ci.yml

```
@@ -0,0 +1,64 @@
+name: "schemacode_ci"
+
+on:
+  push:
+    branches:
```

```suggestion
    paths:
      - 'tools/schemacode/**'
      - 'src/schema/**'
    branches:
```
and similar one for `pull_request` to not bother testing the package (if takes long, didn't check) if no package related changes?

---

Ohhh... I didn't know that was an option! The tests are currently pretty fast, but I was worried about spamming PRs with a bunch of unnecessary checks. This should deal with that nicely.